bvnsMakefile

```diff
@@ -0,0 +1,21 @@
+LSOURCES= elooplib/eventloop.cpp elooplib/server.cpp elooplib/client.cpp elooplib/node.cpp  
+SOURCES = bvnServer/main.cpp
+HEADERS = ./elooplib
+
+all: bvnserver
+
+bvnserver: main.o eventloop.a
+	g++ -std=c++17 main.o eventloop.a -o bvnserver
+
+main.o: $(SOURCES)
+	g++ -c -std=c++17 -c -I./elooplib $(SOURCES) 
+
+eventloop.a: eventloop.o
+	ar rcs eventloop.a 
+	
+eventloop.o: $(LSOURCES)
+	g++ -c -std=c++17  -c $(LSOURCES)
```

Comment:
Wrong, each cpp file must have it's own .o file, this way you won't need to recompile every source file once you change the library.

Try `touch`ing some cpp file and invoking `make VERBOSE=1` to see how make reacts to incremental changes